test/generators.test.js

```diff
@@ -51,7 +51,7 @@ describe('generator-feathers', function() {
       });
   }
 
-  before(() => helpers.run(path.join(__dirname, '..', 'generators', 'app'))
+  beforeEach(() => helpers.run(path.join(__dirname, '..', 'generators', 'app'))
```

Comment:
I needed to switch this to `beforeEach`. For some reason when running the tests using the same app and same directory some config values aren't being cleaned up properly. Potentially this is due to it being headless but I went and verified manually that you can generate multiple services with different dbs and different adapters on the same app. It even works if you say, used mongodb and then decided to switch to mongoose.